query/src/main/scala/filodb/query/exec/PeriodicSamplesMapper.scala

```diff
@@ -38,12 +38,12 @@ final case class PeriodicSamplesMapper(start: Long,
   protected[exec] def args: String =
     s"start=$start, step=$step, end=$end, window=$window, functionId=$functionId"
 
-  //scalastyle:off
+ //scalastyle:off
```

Comment:
Could this scalastyle be either made more specific or better yet removed?  Sorry don't remember if it was already there or not

Reply:
This is also not the latest file. I have added ``` //scalastyle:off
 // Method is longer than 50 lines``